apps/dashboard/app/(app)/apis/[apiId]/settings/default-bytes.tsx

```
@@ -64,43 +69,50 @@ export const DefaultBytes: React.FC<Props> = ({ keyAuth }) => {
+        <Card>
+          <CardHeader>
+            <CardTitle>Default Bytes</CardTitle>
+            <CardDescription>
+              Set default Bytes for the keys under this API. Default byte size must be between{" "}
+              <span className="font-bold">8 to 255</span>
+            </CardDescription>
+          </CardHeader>
+          <CardContent>
+            <div className="flex flex-col space-y-2">
+              <input type="hidden" name="keyAuthId" value={keyAuth.id} />
+              <label className="hidden sr-only">Default Bytes</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)